lib/response/headers.ex

```diff
@@ -1,6 +1,13 @@
 defmodule HTTPServer.Response.Headers do
   alias HTTPServer.Request
-  defstruct content_length: nil, content_type: nil, host: nil, location: nil, allow: nil
+
+  defstruct content_length: nil,
+            content_type: nil,
+            host: nil,
+            location: nil,
+            allow: nil,
+            link: nil
```

Comment:
I do not think the link is needed anymore? so there would be no changes to this file?